client/src/utils/api.js

```diff
@@ -19,9 +19,14 @@ export async function getProjectById(id) {
   const project = await response.json();
   return project;
 }
+export async function getProjectByTag(tag) {
+  const response = await fetch(`/api/browse/${tag}`);
+  const project = await response.json();
+  return project;
+}
 
 export async function getCategory(category) {
-  const response = await fetch(`/api/projects?data.category=${category}`);
+  const response = await fetch(`/api/category/${category}`);
```

Comment:
I would expect to have `categories` instead of `category`

```suggestion
  const response = await fetch(`/api/categories/${category}`);
```